jetson/nav/main.cpp

```diff
@@ -111,6 +121,7 @@ int main()
     lcmObject.subscribe( "/radio", &LcmHandlers::radioSignalStrength, &lcmHandlers );
```

Comment:
any reason why you guys used the config file above to reference the lcm channels and here they're just referenced by name?